test/java/org/chromium/net/CronetUrlRequestTest.java

```diff
@@ -73,20 +93,22 @@ public class CronetUrlRequestTest {
   public void setUp() {
     mTestFramework = mTestRule.startCronetTestFramework();
     assertTrue(NativeTestServer.startNativeTestServer(getContext()));
-    // Add url interceptors after native application context is initialized.
```

Comment:
Consider splitting the test class into multiple ones where each class has a common setUp() / tearDown() step. It's verbose and error prone having to state startCronetEngineForMockUrlRequestJobFactory / shutdown in each test case.

Reply:
I made a different change. Rather than splitting (I wasnt sure what to name the second test: CronetUrlRequestSecondTest?), I modified the test to use a single engine, adding the filter if not testing java.

Let me know if that's fine or if splitting the class is better

Reply:
You're still calling  startCronetEngineForMockUrlRequestJobFactory(); / mMockUrlRequestJobFactory.shutdown(); from a lot of test cases manually which is my primary concern. Is there anything those tests have in common? If so, you can make that attribute part of the test class name (e.g. if all the relevant test cases are related to errors, CronetUrlRequestErrorsTest).

That said I'm happy to live with the change as-is.

Reply:
Hi Dan, you commented on the old version of the file. It should've said outdated but I don't know what happened. Clicking into the file from this comment shows the previous version.

To see the new changes, you have to click on Files Changed above and scroll to the file. It took me a min to figure out why my changes seemed not to be showing.